#BE-3106 Publish Profile Filter for Statuses (#551)
appcircleio/appcircle-docusaurus: docs/publish-module/binary-management/index.md

:::

This streamlined approach ensures that you are always informed of your application's status, simplifying management and enhancing your deployment strategy directly from the Appcircle dashboard.

### Filter By Status

> Within the Publish module where your Publish profiles are listed, you can use the filter option to display specific Publish profiles according to their latest store status.

Reviewer: ```suggestion
In the Publish module, where your Publish profiles are listed, you can use the filter option to display specific Publish profiles based on their latest store status.
```
